jest.config.js

```diff
@@ -37,6 +37,8 @@ const config = {
     '^react-instantsearch-(.*)$':
       '<rootDir>/packages/react-instantsearch-$1/src/',
     '^instantsearch.js$': '<rootDir>/packages/instantsearch.js/src/',
+    '^instantsearch.js/es(.*)$': '<rootDir>/packages/instantsearch.js/src$1',
```

Comment:
something here causes jest to resolve wrong in the ReactDOMServer tests only it seems, not sure why.

Reply:
solved since we no longer have React InstantSearch dom :D